<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>467</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu Etalonnage en Temperature</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/LOGO REACTUALISE.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
background-color: rgb(225, 225, 225); 
border: 1px solid Black;
border-radius: 5px;
 margin-top: 1ex; /* leave space at the top for the title */
 }

QRadioButton{
background-color: rgb(225, 225, 225);}


QComboBox {
background-color: rgb(225, 225, 225);
border: 1px solid Black;
border-radius: 5px;
padding: 1px 18px 1px 3px;}

QLineEdit {
     border: 1px solid Black;
     border-radius: 5px;}

</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>561</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>30</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_saisie_etal">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Saisie Etalonnage </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_validation">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Validation / Modification</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_consultation">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Consultation saisies</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_validation_n_doc">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_consultation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButton_annule_remplace">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Annule et Remplace</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_reeimpression">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Reimpression :  nÂ° du document</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_annule_remplace_n_doc">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_reeimp_n_document">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>LOGO_REACTUALISE_petit_format.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_saisie_etal</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_validation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_consultation</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_reeimpression</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_annule_remplace_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_validation_n_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_consultation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>686</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_annule_remplace</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_reeimp_n_document</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
